torch/distributed/rendezvous.py

```diff
@@ -139,7 +139,9 @@ def _error(msg):
     world_size = int(query["world_size"])
     start_daemon = rank == 0
     assert result.hostname is not None
-    store = TCPStore(result.hostname, result.port, world_size, start_daemon, timeout)
+    store = TCPStore(  # type: ignore[call-arg]
+        result.hostname, result.port, world_size, start_daemon, timeout, multi_tenant=True
```

Comment:
Same question - does it make more sense to flip it to multitenant in the PR that enables it? Although shouldn't really make a difference if you're planning to land the whole stack at around the same time.